src/Acuminator/Acuminator.Vsix/Resources/VSIXResource.resx

```diff
@@ -226,4 +228,10 @@
   <data name="Setting_UseBqlOutlining_Description" xml:space="preserve">
     <value>Collapse parts of BQL commands (only for Roslyn coloring).</value>
   </data>
+  <data name="Setting_UseBqlDetailedOutlining_Title" xml:space="preserve">
+    <value>Outline parts of the BQL</value>
+  </data>
+  <data name="Setting_UseBqlDetailedOutlining_Description" xml:space="preserve">
+    <value>Allow to collapse parts of the BQL (only for Roslyn coloring). If this setting is set to false the Acuminator will collapse only entire BQL queries.</value>
+  </data>
```

Comment:
Collapse parts of BQL commands (only for Roslyn coloring).  If set to false, BQL commands are collapsed only entirely.